tests/smoke.test.js

```diff
@@ -8,7 +8,8 @@ describe('smoke test', () => {
     it('should wait for elements if not found immediately', () => {
         browser.waitForElementScenario()
         const elem = $('elem')
-        assert.equal(elem.isDisplayed(), true)
+        //Element will be found
+        assert.doesNotThrow(() => elem.click())
```

Comment:
This is now a complete different test, we should have two test cases for them

Reply:
Ok. I added a negative test to ensure that isDisplayed returns false on a non existent element.